includes/service-providers/constant_contact/class-newspack-newsletters-constant-contact.php

```diff
@@ -1196,11 +1196,19 @@ public function add_contact( $contact, $list_id = false ) {
 		if ( isset( $contact['metadata'] ) ) {
 			$data['custom_fields'] = [];
 			foreach ( $contact['metadata'] as $key => $value ) {
-				$data['custom_fields'][ strval( $key ) ] = strval( $value );
+				// Update request will fail if metadata contains an empty value, so we set these to '-'.
```

Comment:
As the comment notes, update requests fail with the following error: `Value can't be blank`. So I'm replacing empty string here with `-`. Alternatively I can remove these values from the `custom_fields` array altogether, but that would need to involve checking the existing metadata to verify a value that currently exists isn't intentionally being removed.

Let me know which we prefer!